migrations/schema.sql

```diff
@@ -140,7 +137,7 @@ CREATE TABLE IF NOT EXISTS `pulls` (
   `difficulty` int(11) DEFAULT NULL,
   PRIMARY KEY (`repo`,`number`),
   KEY `pulls_state` (`state`),
-  KEY `pulls_repo` (`repo_owner`,`repo_name`),
+  KEY `pulls_repo` (`repo_owner`),
```

Comment:
It looks like the migration changes from the first one were not added. This is supposed to be a key on column `repo`, not `repo_owner`, right?